app/decorators/trigger_at_decorator.rb

```diff
@@ -1,8 +1,12 @@
 module TriggerAtDecorator
-  def formatted_trigger_time(bold = true)
+  def formatted_trigger_time(bold = true, format=nil)
     return placeholder unless object.trigger_at
 
-    trigger_on = object.trigger_at.strftime("%-d %b %Y")
+    if format.present? && format == "full"
+      trigger_on = object.trigger_at.strftime("%-d %B %Y,")
```

Comment:
I don't think comma is intentional. 

```suggestion
      trigger_on = object.trigger_at.strftime("%-d %B %Y")
```

Reply:
Also changing the month format from Jul to July

Reply:
Refactored trigger_at_decorator